learningpath-api/src/main/scala/no/ndla/learningpathapi/model/api/NewLearningStepV2DTO.scala

```diff
@@ -19,7 +19,9 @@ case class NewLearningStepV2DTO(
     @description("The descriptions of the learningstep") description: Option[String],
     @description("The chosen language") language: String,
     @description("The embed content for the learningstep") embedUrl: Option[EmbedUrlV2DTO],
-    @description("Determines if the title of the step should be displayed in viewmode") showTitle: Boolean,
+    @description(
+      "Determines if the title of the step should be displayed in viewmode. Default is false."
+    ) showTitle: Option[Boolean],
```

Comment:
Dersom tingene skal ha en ikke-dynamisk default fallback (som denne tror jeg?), så tror jeg vi kunne gjort det med `@default` for å få det med i doc?

```suggestion
    @description(
      "Determines if the title of the step should be displayed in viewmode. Default is false."
    @default(false)
    ) showTitle: Boolean,
```